tutorials/quantum_oracle/quantum_oracle_tutorial.ipynb

```diff
@@ -0,0 +1,183 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        res ^= x %2 == 1
</pre><p>If I understand correctly, we could have change the == 1 to ==0, and then later omit the X(y) in the main, right? (checking if the number is even instead if its odd).</p><p> </p><p>It's quite minor, mainly my OCD, but might "cleaner" and slightly clearer. I don't have to change it if you don't feel like, but if so, I just would have add clarification on the purpose of the X(y).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>